templates/masterclasses.html

```diff
@@ -52,17 +51,12 @@ <h3 class="p-media-object__title">
                   <li class="p-media-object__meta-list-item--date">
                     <span class="u-off-screen">Date: </span>{{ session.Date.Formatted }}
                   </li>
-                  <!-- Add to calendar button, URL documentation: https://github.com/InteractionDesignFoundation/add-event-to-calendar-docs/blob/main/services/google.md -->
-                  <!-- Currently, all events start at 11AM CST with duration of 45 minutes -->
+                  <!-- Add to calendar btn uses a gcal published event https://support.google.com/calendar/answer/41207 -->
```

Comment:
The linked article seems to be about another topic?

Reply:
Check the second heading in the article: https://support.google.com/calendar/answer/41207#:~:text=Let%20site%20visitors%20save%20a%20calendar%20event

The purpose of linking this article, is to inform reader about what "published event" is, and how it's created. This is the best article I could find by Google that explains anything about a "Published event".